tests/unit/test_docs.py

```
@@ -0,0 +1,73 @@
+        result = subprocess.run(
+            [sys.executable, "-m", "pdoc", "neuraldbg", "--output-dir", str(tmp_path)],
+            capture_output=True,
+            text=True,
+        )
+        assert result.returncode == 0, (
+            f"pdoc failed with exit {result.returncode}:\n{result.stderr}"
+        )
+
+    def test_docs_output_file_created(self, tmp_path):
+        """pdoc produces an HTML file for the neuraldbg module."""
+        subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)